### PR TITLE
Creation-page-formation

### PR DESCRIPTION
**Pull request by yazidsef**
creation front-end page formation

### CODE REVIEW

File: src/Controller/FormationController.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Controller;
+
+class HomeController extends AbstractController
```
Comment (LaserHuy) [inline src/Controller/FormationController.php:5]: HomeController to FormationController

File: src/View/formation/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,68 @@
+{% extends '../layout.html.twig' %}
```
Comment (LaserHuy) [inline src/View/formation/index.html.twig:1]: delete ../

**Review by LaserHuy — CHANGES_REQUESTED** [review]

File: src/View/formation/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,70 @@
+{% extends 'layout.html.twig' %}
+{% block stylesheet %}<link rel="stylesheet" href="/assets/css/formation.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/remixicon/3.4.0/remixicon.css" crossorigin="">
```
Comment (Benja-san) [inline src/View/formation/index.html.twig:3]: this link is already imported in layout remove it and import parent content inside your stylesheet block

File: src/View/formation/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+{% block content %}
+    <main>
+        <!-- debut de section formation description  -->
+        <section class="formation-description grid">
+
+
+            <!-- debut de section formation description  -->
+            <div class="formation-description-text">
+                <h2>&quot; formation PHP developer &quot;</h2>
```
Comment (Benja-san) [inline src/View/formation/index.html.twig:14]: H1 before H2

**Review by LaserHuy — APPROVED** [review]
ok its good

File: database.sql [hunk 5]
```diff
@@ -169,7 +169,7 @@ ALTER TABLE `Student_Reviews`
 -- Contenu de la table `Discipline`
 --
 
-INSERT INTO 'Discipline' ('id', 'icon', 'name', 'description') VALUES
+INSERT INTO `Discipline` (`id`, 'icon', 'name', 'description') VALUES
```
Comment (Benja-san) [inline database.sql:172]: you don't need to insert id values since it is in auto increment as a primary key

File: src/View/formation/index.html.twig
Comment (Benja-san) [inline src/View/formation/index.html.twig:1]: change H3 into h2 then

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — CHANGES_REQUESTED** [review]
databse changes haven't been done

**Review by Benja-san — APPROVED** [review]

**Pull request merged**